### DISCUSSION TITLE
Why can't I use a capitalized table name in my posgres trigger function?

### DISCUSSION THREAD

**Discussion by lumenwrites** [supabase/supabase#7095] [Questions] [answered]
Hi! I'm trying to set up my own public `Profile` table that holds custom data for my users. Every time a user signs up with Supabase Auth and is INSERTed into `auth.users` table, I want to run a trigger function that adds the user to my `Profile` table.

Everything works well when my table is called `profile` and the function looks like this:
```
begin
  insert into public.profile(id, email)
  values(new.id, new.email);

  return new;
end;
```

But now I want to capitalize the profile table name. I rename the `profile` table into `Profile`, and change the function to this:
```
begin
  insert into public.Profile(id, email)
  values(new.id, new.email);

  return new;
end;
```

And it stops working, I'm getting an error:
> relation "public.profile" does not exist`

Do you know what am I doing wrong? It's like postgres doesn't see that I have capitalized the table name when I'm referring to it in my postgres function.

**Comment by GaryAustin1** [comment, marked as answer]
In general with Postgres you should use lower case unless you have really good reason not to (Prisma, etc).
If you use upper case for table or column names you HAVE to put double quotes around them "TableName" when you reference them, or the name you use gets converted to lowercase and won't match anything.

**Reply by lumenwrites** [reply]
Yep, I'm using Prisma. Thank you so much!

This syntax looks a bit weird, but it worked!

```
begin
  insert into public."Profile"(id, email)
  values(new.id, new.email);

  return new;
end;
```

**Reply by absolutegravitas** [reply]
which genius at postgres thought that was a good idea....

**Reply by lixaotec** [reply]
Can´t belive this quotes hell postgres adopted.

Is ther any way to disable it? like SQL Server does?

**Reply by GaryAustin1** [reply]
There is not a way to "disable" the need for double quotes besides not using capital letters.
